assets/scripts/dialogs/UpgradeDialog.scss

```diff
@@ -12,15 +11,76 @@
     [dir="rtl"] & {
       text-align: right;
     }
-  }
 
-  .dialog-content {
-    overflow: hidden;
-    overflow-y: scroll;
+    [role="subtitle"] {
+      color: #1b914e;
+    }
   }
 }
 
 .upgrade-dialog-content {
   display: flex;
   margin-bottom: 2em;
 }
+
+.upgrade-dialog-left {
+  flex-basis: 240px;
+  margin-right: 40px;
+  border-right: 1px solid #cde1ea;
+  padding-right: 40px;
+  padding-bottom: 16px;
+  position: fixed;
+  width: 280px;
+
+  [dir="rtl"] & {
+    margin-left: 40px;
+    margin-right: 0;
+    border-left: 1px solid #cde1ea;
+    border-right: 0;
+    padding-left: 40px;
+    padding-right: 0;
+  }
+
+  p:first-child {
+    margin-top: 0;
+  }
+
+  p {
+    margin-top: 1.5em;
+  }
+
+  ul {
+    list-style: none;
+    padding: 0;
+    margin: 0;
+  }
+
+  li {
+    margin: 16px 0px;
```

Comment:
This is throwing a lint error because it expects the `0` to be unitless (e.g. `0` instead of `0px`).

Reply:
Are our lint checks different? I didn't get a commit lint check issue with this

Reply:
Hmm. Well, the CI linting should be the same as the checks happening locally. It was triggered here:
https://github.com/streetmix/streetmix/pull/2325/checks?check_run_id=2398180113

However, local linting usually only checks recently changed files unless you run `npm run lint` specifically; it might have snuck in and didn't get get linted.